profiles/community/init.sh

```diff
@@ -0,0 +1,14 @@
+# #!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+source /src/galaxy_ng/profiles/base/init.sh
```

Comment:
Don't run the init script from the base profile. You can make galaxy_ng/base a required profile for this profile by doing this: https://github.com/ansible/galaxy_ng/blob/master/profiles/insights/profile_requirements.txt. That will cause oci-env to exit if the base profile isn't included in your compose profile.

Reply:
I'm doing this so I can use the `setup_collection_signing_service` command, without this import, the command doesn't exist.

Reply:
if the signing service is required, couldn't you just set the ENABLE_SIGNING env var in the pulp_config.env file?